imports/api/activities/activity.js

```diff
@@ -40,15 +34,15 @@ Schemas.Activity = new SimpleSchema({
     endDate: {type: String},
     endTime: {type: String},
     capacity: {type: SimpleSchema.Integer},
-    isRange: {type: Boolean},
+    isRange: {type: Boolean, optional: true},
     conflict: {type: String, optional: true},
 
     attendees: {type: Array, optional: true},
     'attendees.$': {type: new SimpleSchema({
       email: Schemas.Email,
       firstName: {type: String},
       lastName: {type: String},
-      numberOfPeople: {type: SimpleSchema.Integer},
+      numberOfPeople: {type: Number},
```

Comment:
in simpl-schemas for Number's we use SimpleSchema.Integer

Reply:
oh right. I don't know why I even changed it..